connector/src/main/java/org/geysermc/connector/network/translators/bedrock/entity/player/BedrockActionTranslator.java

```diff
@@ -56,6 +57,11 @@
 
 @Translator(packet = PlayerActionPacket.class)
 public class BedrockActionTranslator extends PacketTranslator<PlayerActionPacket> {
+    private final boolean cacheChunks;
```

Comment:
What's the point of this boolean here? Why can't you just `GeyserConnector.getInstance().getConfig().isCacheChunks()` in place of everywhere it is used?

Reply:
Historically, we've done this in the past, so I've been following the practice. I think the idea is that having a local variable is slightly better on performance? See JavaTileEntityTranslator and JavaChunkDataTranslator.

Reply:
Doesn't seem necessary to me honestly, probably just worth removing it here for now and in a future PR when the chunk and tile entity translators are modified, updating those accordingly. As for performance, this should not really impact it at all.

Reply:
Will do now.